### PR TITLE
chore: add package placeholder

### PR DESCRIPTION
**Pull request by nejch**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
+    - name: Install poetry
+      run: python3 -m pip install --user poetry==1.3.1
+    - name: Publish
+      env:
+        POETRY_HTTP_BASIC_PYPI_USERNAME: __token__
+        POETRY_HTTP_BASIC_PYPI_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```
Comment (nejch) [inline .github/workflows/publish.yml:21]: I've set up an initial token in secrets for now. As soon as the first version is published, we can set up a new one scoped to just the project.

**Comment by nejch** [comment]
@fgreinacher could you take a look? It's just a very basic setup to reserve a placeholder on PyPI like we did for https://pypi.org/project/siemens. We'll probably want to switch to semantic-release for the proper package, but just to have some basic infra set up here.

I tested a local publish with test.pypi.org, see https://test.pypi.org/project/mkdocs-siemens/.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (fgreinacher) [reply .github/workflows/publish.yml:21]: Sounds good 👍

**Review by fgreinacher — APPROVED** [review]

**Comment by fgreinacher** [comment]
Thanks!

**Pull request merged**